### PR TITLE
Update license and appveyor

### PR DESCRIPTION
**Pull request by torn4dom4n**
* Using nodejs 6
* Remove npm WARN

### CODE REVIEW

**Comment by 50Wliu** [comment]
This changes the license from GPL2 to MIT.  Please do not do that.
The issue and PR templates were not created in this repo for a reason.

**Comment by torn4dom4n** [comment]
@50Wliu I added COYPING and removed issue and PR templates!

File: package.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
-  "name": "language-puppet",
   "version": "0.22.0",
+  "name": "language-puppet",
+  "description": "Puppet language support in Atom",
+  "license": "MIT",
```
Comment (50Wliu) [inline package.json]: This is still stating that the license is MIT, when it's GPL2.

Comment (torn4dom4n) [inline package.json]: Done!

**Comment by 50Wliu** [comment]
Sorry, looks like I wasn't specific enough.  The license identifier needs to be `GPL-2.0`.

**Pull request merged**